server/pkg/integration/integration.go

```diff
@@ -71,7 +71,7 @@ func (i *Integration) SetDeveloper(developer UserID) {
 	i.developer = developer
 }
 
-func (i *Integration) Webhook() []*Webhook {
+func (i *Integration) Webhooks() []*Webhook {
 	return i.webhook
```

Comment:
```suggestion
	return slices.Clone(i.webhooks)
```

rename `webhook` filed to `webhooks`